src/pages/patientView/trialMatch/TrialMatchTable.tsx

```diff
@@ -306,6 +327,13 @@ export default class TrialMatchTable extends React.Component<ITrialMatchProps> {
     render() {
         return (
             <div>
+                <p style={{marginBottom: '0'}}>Curated genomic and clinical criteria from open clinical trials at Memorial Sloan Kettering. Please <a href="mailto:team@oncokb.org">contact us</a> or submit <a onClick={() => this.showGeneralFeedback=true}>feedback form</a> if you have any questions.</p>
```

Comment:
I'm surprised that changing `showGeneralFeedback` on the click event would actually work in mobx observable. @alisman what's the situation that a `@autobind` has to be attached?

Reply:
this will work.  if you pass it a function directly (not an anonymous callback) then it has to be bound with autobind.  Using an anonymous callback is discouraged b/c it causes virtual dom to be differed each time. But in this case, it's totally negligible and not worth changing.   So, rule of thumb is, if something is going to be rendered a lot and occurs many times in DOM, then use a bound handler.  If not, do the above.